pkg/runtime/runtime.go

```diff
@@ -160,15 +160,20 @@ func (r *runtime) RunStream(ctx context.Context, sess *session.Session) <-chan E
 			telemetryClient.RecordSessionStart(ctx, r.currentAgent, sess.ID)
 		}
 
+		a := r.team.Agent(r.currentAgent)
+		model := a.Model()
+		modelID := model.ID()
+
+		messages, err := sess.GetMessages(a)
+		if err != nil {
+			events <- Error(fmt.Sprintf("getting messages: %v", err))
+			return
```

Comment:
Not sending back the user message will break the TUI in this case, the TUI only shows the user message when it receives the UserMessage event

Reply:
Show the user message too?